system-tests/lib/system-tests.ts

```diff
@@ -413,18 +412,16 @@ const copy = function () {
 
     debug('Copying Circle Artifacts', ca, videosFolder, screenshotsFolder)
 
+    const copy = (src, dest) => {
+      return fsExtra.copyAsync(src, dest, { overwrite: true }).catch({ code: 'ENOENT' }, () => { })
+    }
+
     // copy each of the screenshots and videos
     // to artifacts using each basename of the folders
-    return Bluebird.join(
-      screenshots.copy(
-        screenshotsFolder,
-        path.join(ca, path.basename(screenshotsFolder)),
-      ),
-      videoCapture.copy(
-        videosFolder,
-        path.join(ca, path.basename(videosFolder)),
-      ),
-    )
+    return Promise.all([
```

Comment:
I wonder why `join` was used here originally. Doesn't look to be any type of handler?

Reply:
:woman_shr